test/__tests__/patterns.js

```diff
@@ -0,0 +1,72 @@
+import { runCli } from "../utils";
+
+// MIGRATION: Worked as is.
+describe("multiple patterns", () => {
+  runCli("patterns", ["directory/**/*.js", "other-directory/**/*.js", "-l"]).test({
+    status: 1,
```

Comment:
This is also odd to me (copied as is from the old CLI). Why do we return 1 when there's no error and the operation was successful?

Reply:
Did we find any matching files?

Reply:
Yup, there are 4 matches written to stdout in this case.
![image](https://github.com/user-attachments/assets/e243885c-d731-4c07-b541-3eb4efe45e30)

Reply:
I don't get it, if both globs are finding files, and they are mentioned in stdout, why are we returning 1 here? 🤔